agent/command.go

```diff
@@ -124,6 +127,16 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				if options.isTaskCommands || options.shouldSetupFail {
 					return errors.Wrap(err, "command failed")
 				}
+				// TODO EVG-6009 remove the following once monitoring is done
+				if options.isPre {
+					grip.Alert(message.Fields{
+						"message": "command failed pre",
+						"ticket":  "EVG-6009",
+						"command": cmd,
```

Comment:
there's a Name() and Type() method on command that might be more useful.

Reply:
I wanted the command params as well though. Is there a good way to log those?

Reply:
not really.